blank-canvas-blocks/assets/navigation.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * File navigation.js.
+ *
+ * Required to open the mobile navigation.
+ */
+(function () {
+
+	function mobalizeNavigationBlock( navMenu ) {
```

Comment:
Sorry, this is not meant to be pedantic :)

```suggestion
	function mobilizeNavigationBlock( navMenu ) {
```